cmake/ngraph.cmake

```diff
@@ -0,0 +1,53 @@
+include(ExternalProject)
+
+add_library(ngraph_interface_lib INTERFACE)
+
+set(EXTERNAL_NGRAPH_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR}/3rdparty/ngraph)
+set(EXTERNAL_NGRAPH_SOURCE_DIR ${CMAKE_SOURCE_DIR}/3rdparty/ngraph)
```

Comment:
Do you want users to be able to override these?  If not, you may wish to use the [`INTERNAL` or `FORCE` options](https://cmake.org/cmake/help/v3.3/command/set.html).

Also, to reduce namespace confusion, you might consider having the variable names _start_ with `NGRAPH_`, or maybe even `MXNET_NGRAPH_`.